### ISSUE TITLE
"Error creating image: [404] Not found" if source image=linode/centos7

### ISSUE THREAD

**Issue by kbinghamnetfoundry** [linode/packer-builder-linode#13]
## General:

* [x] Have you removed all sensitive information, including but not limited to access keys and passwords?
* [x] Have you checked to ensure there aren't other open or closed pull requests for the same bug/feature/question?

----

## Bug Reporting

Possible duplicate of https://github.com/linode/packer-builder-linode/issues/10#issuecomment-477525340

### Expected Behavior
image is created from Linode disk
### Actual Behavior
Linode is shutdown and Packer exits with an error like `==> linode: Error creating image: [404] Not found`
### Steps to Reproduce the Problem

  1. `packer build ./packer/linode-build-marketplace.json`
  1. wait for provisioners to complete without error and Linode to shut down
  1. image create fails with (HTTP?) 404

### Environment Specifications

```
❯ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Pop!_OS 18.04 LTS
Release:        18.04
Codename:       bionic

❯ go version
go version go1.12.4 linux/amd64

❯ packer --version
1.3.5
```
Env var `LINODE_TOKEN` is set and is sourced by Packer to assign `linode_token` in the template. The token has all privileges for the Linode cloud manager account.

#### Screenshots, Code Blocks, and Logs

PACKER_LOG interleaved with output from `packer build`
```2019/04/25 17:18:43 packer-1.3.5: 2019/04/25 17:18:43 shutting down the SSH proxy
2019/04/25 17:18:43 [INFO] (telemetry) ending ansible
==> linode: Shutting down Linode...
==> linode: Creating image...
2019/04/25 17:20:08 ui error: ==> linode: Error creating image: [404] Not found
==> linode: Error creating image: [404] Not found
2019/04/25 17:20:09 [INFO] (telemetry) ending linode
2019/04/25 17:20:09 ui error: Build 'linode' errored: Error creating image: [404] Not found
2019/04/25 17:20:09 Builds completed. Waiting on interrupt barrier...
2019/04/25 17:20:09 machine readable: error-count []string{"1"}
Build 'linode' errored: Error creating image: [404] Not found
2019/04/25 17:20:09 ui error: 

==> Some builds didn't complete successfully and had errors:
2019/04/25 17:20:09 machine readable: linode,error []string{"Error creating image: [404] Not found"}
2019/04/25 17:20:09 ui error: --> linode: Error creating image: [404] Not found
==> Builds finished but no artifacts were created.
2019/04/25 17:20:09 [INFO] (telemetry) Finalizing.
==> Some builds didn't complete successfully and had errors:
--> linode: Error creating image: [404] Not found

==> Builds finished but no artifacts were created.
2019/04/25 17:20:09 waiting for all plugin processes to complete...
2019/04/25 17:20:09 /home/kbingham/Sites/netfoundry/bin/packer-1.3.5: plugin process exited
2019/04/25 17:20:09 /home/kbingham/.packer.d/plugins/packer-builder-linode: plugin process exited
2019/04/25 17:20:09 /home/kbingham/Sites/netfoundry/bin/packer-1.3.5: plugin process exited
```

Packer template `builders[]`
```
    "builders": [
        {
            "type": "linode",
            "linode_token": "{{user `linode_token`}}",
    
            "region": "us-central",
            "swap_size": 256,
            "image": "linode/centos7",
            "instance_type": "g6-nanode-1",
            "instance_label": "packerbuilder-{{timestamp}}",
    
            "image_label": "nf-{{user `img_type`}}-{{user `release_version`}}",
            "image_description": "nf-{{user `img_type`}}-{{user `release_version`}}",
            
            "ssh_username": "{{user `ssh_username`}}"
        }
    ],
```

#### Additional Notes

----

The [Linode Community](https://www.linode.com/community/questions/) is a great place to get additional support.

**Comment by displague** [comment]
As you uncovered @kbinghamnetfoundry, I believe this is related to https://github.com/linode/packer-builder-linode/issues/10#issuecomment-486937744 ..

I recreated your image scenario with the builder above and also encountered the 404.  The 404 is caused by the image not existing after the imaging job fails because of the account limits.

By removing the other images on my account I was able to successfully create this CentOS based image.

As pointed out in #10, the best way to remedy this is to reach out to Linode support about an image quote increase.
